tensorflow/contrib/tensorrt/python/__init__.py

```
@@ -19,7 +19,16 @@
 from __future__ import print_function
 
 # pylint: disable=unused-import,line-too-long
-from tensorflow.contrib.tensorrt.python.ops import trt_engine_op
-from tensorflow.contrib.tensorrt.python.trt_convert import calib_graph_to_infer_graph
-from tensorflow.contrib.tensorrt.python.trt_convert import create_inference_graph
+try:
+  from tensorflow.contrib.tensorrt.python.ops import trt_engine_op
+  from tensorflow.contrib.tensorrt.python.trt_convert import calib_graph_to_infer_graph
+  from tensorflow.contrib.tensorrt.python.trt_convert import create_inference_graph
+except:
+  no_trt_message = (
+      '**** Failed to initialize TensorRT. This is either because the TensorRT'
+      ' installation path is not in LD_LIBRARY_PATH, or because you do not have it'
+      ' installed. If not installed, please go to'
+      ' https://developer.nvidia.com/tensorrt to download and install'
+      ' TensorRT ****''')
+  print(no_trt_message)
```

We need to raise the exception after printing, otherwise it'll pass through.